addons/website_blog/tests/test_menu.py

```diff
@@ -0,0 +1,35 @@
+import odoo.tests
+
+@odoo.tests.tagged('post_install', '-at_install')
+class TestWebsiteBlogMenu(odoo.tests.HttpCase):
```

Comment:
Ideally this should have been part of the `test_website` module which is done for this purpose: not testing website behavior on website bridge module just because we have no way to test it in website.

I think it would be fairly quick to move it there if you don't mind :grimacing: 

Sorry to be annoying..

Reply:
You mean that you want me to create a new route in this module that render a view with those `test.model `records ?
Maybe simpler to keep the test as it is and move it in test_website_module no ?
> This module contains tests related to website modules. It allows to test website business code when another website module is installed.

Reply:
`test_website_module` is not done for this, it's for stuff like configurator where you need all modules to be installed to test the flow, since a module cannot be installed inside a test.
And it was not possible to create fake data in test_website to test it for those same reason (it would require the test in test_website to install something)

> You mean that you want me to create a new route in this module that render a view with those test.model records?

That's the idea yes, we already have one template doing t-field on test.model field: `model_item` which is doing `<div class="col" t-field="record.name"/>`

It's called by this route:
```py
@http.route(['/test_website/model_item/<int:record_id>'], type='http', methods=['GET'], auth="public", website=True, sitemap=False)
    def test_model_item(self, record_id):
        values = {
            'record': request.env['test.model'].sudo().browse(record_id),
        }
        return request.render("test_website.model_item", values)
```
        
You can maybe simply reuse this route?